ui/src/components/input/__tests__/use-mask.cy.js

```diff
@@ -1,21 +1,116 @@
+import QInput from '../QInput'
+
+function getHostElement () {
+  return cy.get('.q-field')
+}
+
+function mountQInput (options) {
+  return cy.mount(QInput, options)
+}
+
 describe('use-mask API', () => {
   describe('Props', () => {
     describe('Category: behavior', () => {
       describe('(prop): mask', () => {
-        it.skip(' ', () => {
-          //
+        const masks = [
```

Comment:
Keep the masks declaration outside the `describe` and cycle inside the test itself, as we did for other similar scenarios

Unless you have some reasons to handle it in this way